src/test/java/ttt/CellTest.java

```diff
@@ -0,0 +1,83 @@
+package ttt;
+
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+import static org.hamcrest.Matchers.*;
```

Comment:
`import org.hamcrest.Matchers` and `import static org.hamcrest.Matchers.*` can be removed as they are not being used.